### ISSUE TITLE
Default AWS provider - worker nodes fail to join cluster

### ISSUE THREAD

**Issue by benpriestman** [kubicorn/kubicorn#648]
```
$kubicorn version
{"Version":"master","GitCommit":"","BuildDate":"2018-07-02 12:21:28.951420752 +0000 UTC","GoVersion":"go1.10","GOOS":"linux","GOArch":"amd64"}
~/go/src/github.com/kubicorn/kubicorn
$git rev-parse HEAD
5914f8b1ee8beb76b1a64a28a44203f5b1d4becd
```

Trying out the `aws` provider, no worker nodes are brought up.

Looking at the cloud-init logs on the worker node, I see:
```
[preflight] Running pre-flight checks.
        [WARNING FileExisting-crictl]: crictl not found in system path
Suggestion: go get github.com/kubernetes-incubator/cri-tools/cmd/crictl
[preflight] Starting the kubelet service
discovery: Invalid value: "": using token-based discovery without DiscoveryTokenCACertHashes can be unsafe. set --discovery-token-unsafe-skip-ca-verification to continue
Cloud-init v. 17.1 running 'modules:final' at Mon, 02 Jul 2018 12:13:44 +0000. Up 10.04 seconds.
2018-07-02 12:14:15,421 - util.py[WARNING]: Failed running /var/lib/cloud/instance/scripts/part-001 [3]
2018-07-02 12:14:15,423 - cc_scripts_user.py[WARNING]: Failed to run module scripts-user (scripts in /var/lib/cloud/instance/scripts)
2018-07-02 12:14:15,424 - util.py[WARNING]: Running module scripts-user (<module 'cloudinit.config.cc_scripts_user' from '/usr/lib/python3/dist-packages/cloudinit/config/cc_scripts_user.py'>) failed
Cloud-init v. 17.1 finished at Mon, 02 Jul 2018 12:14:15 +0000. Datasource DataSourceEc2Local.  Up 40.74 seconds
```
Adjusting the launch configuration' s user data to use the switches used in https://github.com/kubicorn/bootstrap/blob/0602657f00474e8e4ec34d29aaf504463315ebbd/amazon_k8s_debian_stretch_node.sh#L52 and respawning a node, I now have a worker node and can deploy a test application.

If I've understood things correctly, the bootstrap scripts are pulled from github by kubicorn, not read from disk, so I can't really test the effect of making this change in-code. Nor am I sure whether it is desirable to bypass ca cert checking. I don't see an open issue relating to it.

The amazon_k8s_centos_7_node.sh bootstrap script is also missing this parameter.

**Comment by xmudrii** [comment]
Hi there!

Thank you so much for reporting the issue! I have merged kubicorn/bootstrap#17 and kubicorn/bootstrap#18 which should fix the issue. If problems persists, re-open the issue or create a new one.

About the local scripts, it's possible to use scripts from your local machine by setting the `KUBICORN_FORCE_LOCAL_BOOTSTRAP` environment variable, such as: `KUBICORN_FORCE_LOCAL_BOOTSTRAP=1 kubicorn apply mycluster -v 4`.

Thanks once again for reporting the issue!

**Comment by benpriestman** [comment]
Wow! That was fast!
Thanks for the info on the FORCE_LOCAL flag.
I've got some general questions about using profiles, but they might be better on Slack.
